api/src/backend/api/filters.py

```diff
@@ -426,6 +426,16 @@ def maybe_date_to_datetime(value):
         return dt
 
 
+class FindingDynamicFilter(FilterSet):
+    updated_at = DateFilter(field_name="updated_at", lookup_expr="date")
```

Comment:
We should filter by `inserted_at`, not `updated_at`. Findings are never updated and it does not make much sense.

Also, you can define `required=True` here to prevent validation on view level and add the field and required tag in the documentation without further changes.